lib/presentation/mixin/send_files_mixin.dart

```diff
@@ -47,7 +47,7 @@ mixin SendFilesMixin on ImagePickerMixin {
     final sendFileInteractor = getIt.get<SendFileInteractor>();
     Navigator.pop(context);
     final result = await FilePicker.platform.pickFiles(
-      withData: true,
+      withReadStream: true,
```

Comment:
IMO, it is not safety, please check more detail in `pickFiles` method

Reply:
With withReadStream is true, the FilePicker will create a file and openRead() for it. So, I think it's safety.
<img width="490" alt="Screenshot 2023-08-03 at 00 47 00" src="https://github.com/linagora/twake-on-matrix/assets/43041967/e27e4e73-44f0-4a02-82d6-e9cec92fd3c8">

Reply:
@sherlockvn Pls create a small demo prove it right

Reply:
Done.

https://github.com/linagora/twake-on-matrix/assets/43041967/5f8c4c39-39d8-40a1-8ab1-41826f39ea44